lib/extractor/index.ts

```diff
@@ -196,7 +197,14 @@ function layersWithLatestFileModifications(
 
       // not adding the original file to extractedLayers
       // and removing it from the set since it can be found in consecutive layers
-      if (removedFilesToIgnore.has(filename)) {
+      if (
+        removedFilesToIgnore.has(filename) ||
```

Comment:
> Would it make sense to have a separate set for folders instead of using removedFilesToIgnore?
Ideally yes, but there is no simple way to know whether a file or folder was deleted at the point where we add it to the set.

I didn't find any other references to that set in the code, so I assume not.